packages/docusaurus-plugin-content-blog/src/pluginOptionSchema.ts

```diff
@@ -64,7 +63,7 @@ export const PluginOptionSchema = Joi.object({
     .allow('')
     .default(DEFAULT_OPTIONS.blogDescription),
   blogSidebarCount: Joi.alternatives()
-    .try(Joi.equal('ALL').required(), Joi.number().required())
+    .try(Joi.equal('ALL').required(), Joi.number().integer().min(0).required())
```

Comment:
Can we have a test to ensure 0 is included? Even the Joi doc is not clear about it but we use 0 to disable the sidebar so we should rather ensure it remains supported over time: `By setting `blogSidebarCount: 0`, the sidebar will be completely disabled`

Reply:
There's already a testcase `'should accept 0 sidebar count'`